src/bin/sol-fbp-generator/type-store.c

```diff
@@ -384,6 +400,73 @@ parse_range_default_value(struct sol_json_scanner *s, struct option_description
 }
 
 static bool
+parse_rgb_default_value(struct sol_json_scanner *s, struct option_description *o)
```

Comment:
minor: this function is pretty similar to 'parse_range_default_value()', you could try to find a way to reuse this similar code.